executable/resource/extract-resource-via-kernel32-functions.yml

```diff
@@ -28,4 +28,3 @@ rule:
             - api: LdrFindResourceEx_U
           - api: kernel32.SizeofResource
           - api: kernel32.FreeResource
-      - api: user32.LoadString
```

Comment:
I don't think that's really what we are after normally. Anyone thing it's worth creating a new rule for it?

Reply:
agree that this should be removed